lighthouse-core/config/seo.js

```diff
@@ -7,6 +7,15 @@
 
 module.exports = {
   extends: 'lighthouse:default',
+  passes: [{
+    passName: 'seoPass',
```

Comment:
if you set `passName` to 'defaultPass' then the gatherer should just be added to the list to run, if not it's a bug and I can look into it :)